libmscore/ottava.cpp

```diff
@@ -145,53 +145,61 @@ Sid Ottava::getPropertyStyle(Pid pid) const
       {
       Q_ASSERT(int(OttavaType::OTTAVA_22MB) - int(OttavaType::OTTAVA_8VA) == 5);
 
-      static const Sid ss[24] = {
+      static const Sid ss[] = {
```

Comment:
Oh, dear. Why don't you use std::vector instead to at least avoid the calls of `sizeof` below?

Reply:
because I tried to keep the code changes as small as possible.
And my knowledge about `std::vector`  is... hmmm... limited.
But I can certianly try, if you want me to ;-)

Reply:
`static const std::vector<Sid> ss = {` here?

Reply:
In this case it is a Boy Scout rule (https://www.matheus.ro/2017/12/11/clean-code-boy-scout-rule/).

Something like that. And just `.size()` instead of `sizeof`s.

Reply:
getting rid of that `24` was following that boy scout rule already ;-)